packages/commerce-sdk-react/src/query-client-config.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {DefaultOptions, QueryClientConfig} from '@tanstack/react-query'
+import {RetryValue} from '@tanstack/query-core/build/lib/retryer'
+
+const NUM_OF_RETRIES = 3 // same as React Query's default
+
+// NOTE: ResponseError does not seem to be exported from the isomorphic sdk lib, so I had to mimic it here
+interface ResponseError extends Error {
+    response: Response
+}
+
+const shouldContinueRetries: RetryValue<unknown> = (failureCount, error) => {
+    let shouldContinue = true
+
+    // See https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/src/static/responseError.ts
+    const isResponseError = Boolean((error as ResponseError).response)
+    const isServerError = isResponseError && (error as ResponseError).response.status >= 500
```

Comment:
What about net work Error? Should we retry? The network could be down a few second and come back up after that. In that case retry does make sense

Reply:
@alexvuong where does the network error come from? I mean who would be the one throwing this kind of error?